src/test/java/uk/gov/hmcts/reform/civil/bpmn/CreateClaimTest.java

```diff
@@ -70,15 +73,17 @@ public CreateClaimTest() {
     @Nested
     class PostFlowStateRename {
 
-        @Test
-        void shouldSuccessfullyCompleteCreateClaim_whenPaymentWasSuccessful() {
+        @ParameterizedTest
+        @ValueSource(strings = {"true", "false"})
+        void shouldSuccessfullyCompleteCreateClaim_whenPaymentWasSuccessful(Boolean rpaContinuousFeed) {
             //assert process has started
             assertFalse(processInstance.isEnded());
 
             //assert message start event
             assertThat(getProcessDefinitionByMessage(MESSAGE_NAME).getKey()).isEqualTo(PROCESS_ID);
 
             VariableMap variables = Variables.createVariables();
+            variables.put(FLOW_FLAGS, rpaContinuousFeed ? Map.of("RPA_CONTINUOUS_FEED", true) : Map.of());
```

Comment:
it should be `variables.put(FLOW_FLAGS, Map.of("RPA_CONTINUOUS_FEED", rpaContinuousFeed)` because the flag is always set (and its value is what's currently set on launch darkly)

Reply:
I need to verify this, as the suggested change is executing the rpa task for false case as well in unit tests.